mod_tariffs.asciidoc

```diff
@@ -65,15 +65,15 @@ Fetch information about all Tariffs.
 
 Endpoint structure definition:
 
-`{tariffs_endpoint_url}?[date_from={date_from}]&amp;[date_to={date_to}]&[offset={offset}]&[limit={limit}]`
+`{tariffs_endpoint_url}?[date_from={date_from}]&amp;[date_to={date_to}]&amp;[offset={offset}]&amp;[limit={limit}]`
```

Comment:
I guess you are asking why I changed `&` to `&amp;`? For consistency, mostly. But also because, at least in VS Code, the syntax highlighting is better with `&amp;` which makes reading the query parameters easier.

Reply:
PDF generations is not 100% correct with `&amp;` So I propose to change them all to `&`. I can to that after the pull request is merged.

Reply:
I fixed it in `mod_tariffs.asciidoc` but will let you do the other documents!